src/point.rs

```diff
@@ -185,6 +186,30 @@ impl Point {
     pub fn is_nan(self) -> bool {
         self.x.is_nan() || self.y.is_nan()
     }
+
```

Comment:
I don't think these belong in the public API. There's also a naming problem, the names suggest that the return value is a `Point` (and the incorrect docstring agrees).